src/harmony/FormEditor/FormEditor.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+
+import { FormControl, FormControlEditor, FormControlError } from '../FormControl/FormControl';
+
+import { FormEditor } from './FormEditor';
+
+const meta: Meta = {
+    title: '@harmony/FormEditor',
+    component: FormEditor,
+};
+
+export default meta;
+
+export const Default: StoryObj<typeof FormEditor> = {
+    args: {
+        outline: true,
+        placeholder: 'Some placeholder...',
+        style: {
+            width: 500,
+        },
+    },
+    render(props) {
```

Comment:
Is not neccessary

Reply:
added `<Layout />` to render instead `style: { width: 500 }`